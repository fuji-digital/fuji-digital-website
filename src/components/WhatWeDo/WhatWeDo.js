import {
  StyledHeader,
  StyledTextContainer,
  StyledBody,
  StyledFujit,
  StyledInformationContainer,
  StyledIconContainer,
  StyledReverseIconContainer,
} from "./WhatWeDo.styles";

import pinkStripeFujit from "../../images/fujits/pink-stripe-fujit.svg";
import purpleTallFujit from "../../images/fujits/purple-tall-fujit.svg";
import pinkRoundShortFujit from "../../images/fujits/pink-round-short-fujit.svg";

import awsIcon from "../../images/icons/aws-icon.svg";
import reactIcon from "../../images/icons/react-js-icon.svg";
import graphqlIcon from "../../images/icons/graphql-icon.svg";
import nodeIcon from "../../images/icons/node-js-icon.svg";

export const WhatWeDo = () => {
  return (
    <StyledBody id="what-we-do">
      <StyledHeader>What We Do</StyledHeader>
      <StyledInformationContainer>
        <StyledTextContainer>
          <StyledFujit src={pinkStripeFujit} />
          <p>
            At our software engineering company, we prioritize delivering
            top-tier work for every website we create. We tailor our solutions
            to meet the unique needs of each client, ensuring high-quality,
            functional, and visually appealing results.
          </p>
        </StyledTextContainer>
        <StyledReverseIconContainer>
          <StyledFujit src={purpleTallFujit} />
          <p>
            We utilize the latest technologies to stay ahead of industry trends.
            By leveraging cutting-edge tools and techniques, we guarantee
            efficient, innovative, and secure websites that perform optimally
            across all devices.
          </p>
        </StyledReverseIconContainer>
        <StyledTextContainer>
          <StyledFujit src={pinkRoundShortFujit} />
          <p>
            Our team prides itself on exceptional customer service and timely
            project completion. We maintain open communication, ensuring clients
            are informed and satisfied throughout the development process.
          </p>
        </StyledTextContainer>
      </StyledInformationContainer>
      <StyledIconContainer>
        <img src={awsIcon} alt="AWS Icon" />
        <img src={reactIcon} alt="React Icon" />
        <img src={graphqlIcon} alt="GraphQL Icon" />
        <img src={nodeIcon} alt="Node Icon" />
      </StyledIconContainer>
      <p>
        Our software stack includes React for dynamic front-end development,
        Node.js for robust back-end solutions, and JavaScript for seamless
        integration. We utilize AWS for reliable cloud services and GraphQL for
        efficient data management. Always eager to innovate, we continuously
        explore and adopt new technologies to enhance our offerings.
      </p>
    </StyledBody>
  );
};
