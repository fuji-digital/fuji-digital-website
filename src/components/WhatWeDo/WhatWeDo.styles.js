import styled from "styled-components";

export const StyledBody = styled.div`
  background: #ebf0f7;
  font-family: "Poppins", sans-serif;
  padding: 1rem;

  p {
    color: #12388f;
    line-height: 1.2;
    padding: 1rem;
    margin: 0%;
  }

  @media screen and (min-width: 768px) {
    padding: 2.5rem;
  }
`;
export const StyledHeader = styled.div`
  color: #12388f;
  font-size: 1.5rem;
  font-weight: bold;
`;

export const StyledInformationContainer = styled.div`
  display: flex;
  flex-direction: column;
  @media screen and (min-width: 1024px) {
    flex-direction: row;
  }
`;

export const StyledTextContainer = styled.div`
  color: #12388f;
  font-size: 1rem;
  display: flex;
  flex-direction: row;
  align-items: center;
`;

export const StyledFujit = styled.img`
  height: 5rem;
`;

export const StyledIconContainer = styled.div`
  display: flex;
  justify-content: space-evenly;
  padding: 1rem 0rem;
  align-items: center;
  img {
    height: 4rem;
    width: 4rem;
  }
  @media screen and (min-width: 768px) {
    padding: 0rem;
  }
`;

export const StyledReverseIconContainer = styled.div`
  display: flex;
  justify-content: space-evenly;
  flex-direction: row-reverse;
  align-items: center;
  img {
    height: 5rem;
    width: 5rem;
  }
  @media screen and (min-width: 768px) {
    flex-direction: row;
  }
`;
