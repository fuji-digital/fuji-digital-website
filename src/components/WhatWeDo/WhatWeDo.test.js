import renderer from "react-test-renderer";
import { WhatWeDo } from "./WhatWeDo";

describe("WhatWeDo", () => {
  it("matches snapshot", () => {
    const component = renderer.create(<WhatWeDo />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
