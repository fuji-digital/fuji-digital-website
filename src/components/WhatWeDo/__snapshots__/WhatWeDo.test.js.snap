// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhatWeDo matches snapshot 1`] = `
<div
  className="sc-aYaIB eGJkdO"
  id="what-we-do"
>
  <div
    className="sc-gEvDqW kHXivm"
  >
    What We Do
  </div>
  <div
    className="sc-eqUzNf jpUvla"
  >
    <div
      className="sc-fqkwJk lmBqrf"
    >
      <img
        className="sc-dcJtft jqAmXX"
        src="pink-stripe-fujit.svg"
      />
      <p>
        At our software engineering company, we prioritize delivering top-tier work for every website we create. We tailor our solutions to meet the unique needs of each client, ensuring high-quality, functional, and visually appealing results.
      </p>
    </div>
    <div
      className="sc-gsFSjX jBnpZW"
    >
      <img
        className="sc-dcJtft jqAmXX"
        src="purple-tall-fujit.svg"
      />
      <p>
        We utilize the latest technologies to stay ahead of industry trends. By leveraging cutting-edge tools and techniques, we guarantee efficient, innovative, and secure websites that perform optimally across all devices.
      </p>
    </div>
    <div
      className="sc-fqkwJk lmBqrf"
    >
      <img
        className="sc-dcJtft jqAmXX"
        src="pink-round-short-fujit.svg"
      />
      <p>
        Our team prides itself on exceptional customer service and timely project completion. We maintain open communication, ensuring clients are informed and satisfied throughout the development process.
      </p>
    </div>
  </div>
  <div
    className="sc-iGgVNO bimDux"
  >
    <img
      alt="AWS Icon"
      src="aws-icon.svg"
    />
    <img
      alt="React Icon"
      src="react-js-icon.svg"
    />
    <img
      alt="GraphQL Icon"
      src="graphql-icon.svg"
    />
    <img
      alt="Node Icon"
      src="node-js-icon.svg"
    />
  </div>
  <p>
    Our software stack includes React for dynamic front-end development, Node.js for robust back-end solutions, and JavaScript for seamless integration. We utilize AWS for reliable cloud services and GraphQL for efficient data management. Always eager to innovate, we continuously explore and adopt new technologies to enhance our offerings.
  </p>
</div>
`;
